thankyou_letters/models/thankyou_config.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import api, models, fields
+
+
+class ThankYouConfig(models.Model):
+    _name = 'thankyou.config'
+    _description = 'Thank You Configuration'
+    _order = "min_donation_amount"
+
+    min_donation_amount = fields.Integer("Minimum Donation Amount",
+                                         help="Amount in the local currency "
+                                              "to apply the configuration")
+    send_mode = fields.Selection('get_send_modes', required=True)
+    thanker_user = fields.Many2one('res.users', string='Thanker')
+
+    @api.multi
+    def for_donation_amount(self, total_amount):
```

Comment:
This method can be removed and replaced by searches.

Reply:
I think it is more complicated because amounts in [0, min amount] won't get matched with the search query you suggested.